templates/todo/projects/csharp-cosmos-sql-func/.vscode/launch.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Web",
+            "request": "launch",
+            "type": "msedge",
+            "webRoot": "${workspaceFolder}/src/web/src",
+            "url": "http://localhost:3000",
+            "sourceMapPathOverrides": {
+                "webpack:///src/*": "${webRoot}/*"
+            },
+        },
+        {
+            "name": "Debug API",
+            "type": "coreclr",
+            "request": "launch",
+            "program": "func",
+            "args": [
+                "host",
+                "start"
+            ],
+            "cwd": "${workspaceFolder}/src/api",
+            "console": "internalConsole",
```

Comment:
`"console": "internalConsole"` and `"stopAtEntry": false` are defaults and do not need to be explicitly added to the configuration. Please remove.

Reply:
I've updated this in latest commit, thank you.